client/src/App.js

```diff
@@ -1,28 +1,12 @@
 import React, { Component } from 'react';
 import logo from './logo.svg';
 import './App.css';
+import FinalGrid from "./components/BeatGrid/FinalGrid"
 
-class App extends Component {
```

Comment:
need to get rid of this in the master in order to get jacks app component in there